.github/workflows/rust.yml

```diff
@@ -125,6 +125,26 @@ jobs:
           command: audit
           args: --ignore RUSTSEC-2020-0071 --ignore RUSTSEC-2022-0040
 
+  spellcheck:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
would be nice to add `Cancel previous workflows`, it takes some time to build this tool

Reply:
It should be cached, though. The latest spellcheck run took 42s in total.